_sass/_docs.scss

```diff
@@ -84,7 +84,8 @@
 .docs h3 code > em,
 .docs h3 > em,
 .docs h4 code > em,
-.docs h4 > em {
+.docs h4 > em,
+.docs li em {
```

Comment:
Looks like there are some other non-platform label elements matching this selector.

I ran 
```js
document.querySelectorAll('.docs li em')
```

on http://electron.atom.io/docs/all/